### PR TITLE
Issue 32 followup

### PR DESCRIPTION
**Pull request by absurdfarce**
A few small follow-up changes to https://github.com/datastax/adelphi/pull/56

### CODE REVIEW

File: docker/cassandra-diff/Dockerfile [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+
+RUN git clone https://github.com/apache/cassandra-diff.git cassandra-diff; \
+	cd cassandra-diff; \
+	git checkout $GIT_ID
+
+WORKDIR /cassandra-diff
+RUN mvn clean package -DskipTests
+
+FROM spark:3.0.0
+
```
Comment (absurdfarce) [inline docker/cassandra-diff/Dockerfile:15]: Only actual change here is the move from "FROM adelphi-dev/spark:3.0.0" to "FROM spark:3.0.0".  Other changes are a result of converting from DOS line break encodings.

**Review by jdonenine — APPROVED** [review]
👍

**Comment by absurdfarce** [comment]
Confirmed changes to Adelphi Python package install via manual runs... all looks good there.

@jdonenine this is ready for another round

**Review by jdonenine — APPROVED** [review]
So much easier now, nice!

**Review by jdonenine — APPROVED** [review]

**Review by grighetto — APPROVED** [review]
LGTM +1

**Pull request merged**